setup.cfg

```diff
@@ -3,4 +3,4 @@ test = pytest
 
 [tool:pytest]
 testpaths = tests
-addopts = -ra
+addopts = -ra --pylint
```

Comment:
Also, might consider running the tests differently, if I just to `pylint semeio` I get a few errors that doesn´t show up when running `pytest --pylint`.

Reply:
>  I would prefer we use something like `tox`, or if we don´t want to make that move now, `pytest.ini` instead.

I completely agree, but was thinking that that was outside the scope of this PR. I just wanted to add `pylint` and did that according to the current test setup in the repo.

Reply:
> Also, might consider running the tests differently, if I just to pylint semeio I get a few errors that doesn´t show up when running pytest --pylint.

Could that be either due to different version of `pylint` or because it does not pick up the `.pylintrc` file correctly? Could you check whether those errors are listed in the config file?

Reply:
> Could that be either due to different version of `pylint` or because it does not pick up the `.pylintrc` file correctly? 

No, don´t think so, I´m using the versions specified in `test_requirements.txt`, and also checked against the `travis` version, which matches.

> Could you check whether those errors are listed in the config file?

Yes, checked that, and none of them are missing in the `.pylintrc`, here is a selection when running `pylint semeio`:

```
************* Module semeio.jobs.rft.utility
semeio/jobs/rft/utility.py:21:0: R0914: Too many local variables (16/15) (too-many-locals)
************* Module semeio.jobs.rft.__init__
semeio/jobs/rft/__init__.py:1:0: C0305: Trailing newlines (trailing-newlines)
************* Module semeio.jobs.rft.zonemap
semeio/jobs/rft/zonemap.py:7:0: C0115: Missing class docstring (missing-class-docstring)
semeio/jobs/rft/zonemap.py:61:22: W1304: Unused format argument 'line_number' (unused-format-string-argument)
semeio/jobs/rft/zonemap.py:61:22: W1304: Unused format argument 'filename' (unused-format-string-argument)
semeio/jobs/rft/zonemap.py:82:8: R1705: Unnecessary "elif" after "return" (no-else-return)
semeio/jobs/rft/zonemap.py:89:8: R1705: Unnecessary "elif" after "return" (no-else-return)
************* Module semeio.jobs.rft.trajectory
semeio/jobs/rft/trajectory.py:7:0: C0115: Missing class docstring (missing-class-docstring)
semeio/jobs/rft/trajectory.py:7:0: R0902: Too many instance attributes (8/7) (too-many-instance-attributes)
semeio/jobs/rft/trajectory.py:34:4: R1710: Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)
semeio/jobs/rft/trajectory.py:61:0: C0115: Missing class docstring (missing-class-docstring)
```
If I remove anything from ´.pylintrc` that shows up as an error.